cypress/integration/onboarding/email/existing_email_adress_spec.js

```diff
@@ -60,6 +56,7 @@ describe('Email Address', () => {
 
         // 8. Logout and signup another user with the same email but different username and password
         cy.apiLogout();
+        cy.clearWhiteListedCookies();
```

Comment:
Per my comment below, I think we should clear the cookies as part of api logout by default.